views/js/pciCreator/ims/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -528,52 +590,62 @@ define([
-                        times:  { label: '&times;',     latex: '\\times',   fn: 'cmd',      desc: 'Multiply' },
-                        divide: { label: '&divide;',    latex: '\\div',     fn: 'cmd',      desc: 'Divide' },
-                        plusminus: { label: '&#177;',   latex: '\\pm',      fn: 'cmd',      desc: 'Plus/minus' },
-                        angle:  { label: '&ang;',       latex: '\\angle',   fn: 'cmd',      desc: 'Angle'},
-                        minus:  { label: '–',           latex: '-',         fn: 'write',    desc: 'Minus'},
-                        plus:   { label: '+',           latex: '+',         fn: 'write',    desc: 'Plus'},
-                        equal:  { label: '=',           latex: '=',         fn: 'write',    desc: 'Equal'},
-                        lower:  { label: '<',           latex: '<',         fn: 'write',    desc: 'Lower than'},
-                        greater: { label: '>',          latex: '>',         fn: 'write',    desc: 'Greater than'},
-                        subscript: { label: 'x&#8336;', latex: '_',         fn: 'cmd',      desc: 'Subscript'},
-                        lbrace: { label: '{',           latex: '{',         fn: 'cmd',      desc: 'Left brace/curly bracket'},
-                        rbrace: { label: '}',           latex: '}',         fn: 'cmd',      desc: 'Right brace/curly bracket'},
-                        lparen: { label: '(',           latex: '(',         fn: 'write',    desc: 'Left parenthesis/round bracket'},
-                        rparen: { label: ')',           latex: ')',         fn: 'write',    desc: 'Right parenthesis/round bracket'},
-                        integral: { label: '&#x222b;',  latex: '\\int',     fn: 'cmd',      desc: 'Indefinite integral'},
-                        timesdot: { label: '·',         latex: '\\cdot',    fn: 'cmd',      desc: 'Times dot'},
-                        triangle: { label: '&#9651;',   latex: '\\triangle',fn: 'cmd',      desc: 'Triangle'},
-                        similar: { label: '&sim;',      latex: '\\sim',     fn: 'cmd',      desc: 'Similar'},
-                        paral:   { label: '&#8741;',    latex: '\\parallel',fn: 'cmd',      desc: 'Is parallel with'},
-                        perp:    { label: '&#8869;',    latex: '\\perp',    fn: 'cmd',      desc: 'Is perpendicular to'},
-                        inmem:   { label: '&isin;',     latex: '\\in',      fn: 'cmd',      desc: 'Is a member of'},
-                        ninmem:  { label: '&notin;',    latex: '\\notin',   fn: 'cmd',      desc: 'Is not a member of'},
-                        union:   { label: '&cup;',      latex: '\\cup',     fn: 'cmd',      desc: 'Set union'},
-                        intersec:{ label: '&cap;',      latex: '\\cap',     fn: 'cmd',      desc: 'Set intersection'}
+                        frac:   { label: self.getLabel('x/y'),
+                            latex: '\\frac',    fn: 'cmd',      desc: 'Fraction' },
```

Comment:
nitpick: it seems there's a formatting issue. Can you please check the formatting is consistent everywhere?